### ISSUE TITLE
Axecore: Ensure elements with ARIA roles have all required ARIA attributes

### ISSUE THREAD

**Issue by haydencordeiro** [shentao/vue-multiselect#1804]
aria-expanded and aria-control attributes missing on multiselect
https://github.com/shentao/vue-multiselect/blob/master/src/Multiselect.vue#L13
